_posts/en/newsletters/2025-08-22-newsletter.md

```diff
@@ -0,0 +1,180 @@
+---
+title: 'Bitcoin Optech Newsletter #368'
+permalink: /en/newsletters/2025/08/22/
+name: 2025-08-22-newsletter
+slug: 2025-08-22-newsletter
+type: newsletter
+layout: newsletter
+lang: en
+---
+This week's newsletter FIXME
+
+## News
+
+- **Draft BIP for block template sharing:** Anthony Towns [posted][towns
+  bipshare] to the Bitcoin-Dev mailing list the [draft][towns bipdraft]
+  of a BIP for how nodes can communicate to their peers the transactions
+  they would attempt to mine in their next block (see [Newsletter
+  #FIXME][news366 templshare]).  This allows the node to share
+  transactions it will accept via its mempool and mining policy that its
+  peers might normally reject by their own policy, allowing those peers
+  to cache those transactions in case they are mined (which improves
+  [compact block relay][topic compact block relay] effectiveness).  The
+  transactions in a node's block template are usually the most
+  profitable unconfirmed transactions known to that node, so peers that
+  previously rejected those transactions for policy reasons might also
+  find them worthy of additional consideration.
+
+  The protocol specified in the draft BIP is simple.  Shortly after a
+  connection with a peer is initiated, the node sends a `sendtemplate`
```

Comment:
```suggestion
  a node initiates a connection with a peer, it sends a `sendtemplate`
```

Reply:
Leaving as-is, either the node or the peer could initiate the connection.  AFAIK, AJ's proposal isn't limited to outbound only.